ManageIQ/manageiq
# Issue a deprecation warning when MiqLdap is being used.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1739073

MiqLdap is being deprecated in the Ivanchuk release (CFME 5.11)

This PR simply issues a deprecation warning message to the audit.log when MiqLdap is being used for authentication.

Steps for Testing/QA [Optional]
-------------------------------

Configure an appliance to use MiqLdap, where authentication mode is set to LDAP or LDAPS.
Attempt to authenticate an LDAP user and confirm the deprecation warning message is
written to the audit.log file
